source/_integrations/alpha_vantage.markdown

```
@@ -80,7 +80,7 @@ foreign_exchange:
 
 ## API information
 
-Alpha Vantage offers two tiers of the API key, one free and one paid for. If you are using the free version, you will be limited to twenty-five queries per day. The sensor polls every 5 minutes, so you will only get data for the first two hours per day. This can be configured using the `scan_interval` variable.
+Alpha Vantage offers two tiers of the API key, one free and one paid for. If you are using the free version, you will be limited to twenty-five queries per day. The sensor polls every 5 minutes, so you will only get data for the first two hours per day. This can be configured using the `scan_interval` variable. To find the necessary symbols, you can use the search function of the API as in this example: https://www.alphavantage.co/query?function=SYMBOL_SEARCH&apikey=<your-api-key&keywords=IBM>
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)